packages/troika-three-text/src/FontParser.js

```diff
@@ -122,38 +122,40 @@ function parserFactory(Typr, woff2otf) {
     return joiningForms
   }
 
-  function stringToGlyphs (font, str) {
+  function stringToGlyphs (font, str, enableLigatures) {
     const glyphIds = []
     for (let i = 0; i < str.length; i++) {
       const cc = str.codePointAt(i)
       if (cc > 0xffff) i++
       glyphIds.push(Typr.U.codeToGlyph(font, cc))
     }
 
-    const gsub = font['GSUB']
-    if (gsub) {
-      const {lookupList, featureList} = gsub
-      let joiningForms
-      const supportedFeatures = /^(rlig|liga|mset|isol|init|fina|medi|half|pres|blws|ccmp)$/
-      const usedLookups = []
-      featureList.forEach(feature => {
-        if (supportedFeatures.test(feature.tag)) {
-          for (let ti = 0; ti < feature.tab.length; ti++) {
-            if (usedLookups[feature.tab[ti]]) continue
-            usedLookups[feature.tab[ti]] = true
-            const tab = lookupList[feature.tab[ti]]
-            const isJoiningFeature = /^(isol|init|fina|medi)$/.test(feature.tag)
-            if (isJoiningFeature && !joiningForms) { //lazy
-              joiningForms = detectJoiningForms(str)
-            }
-            for (let ci = 0; ci < glyphIds.length; ci++) {
-              if (!joiningForms || !isJoiningFeature || formsToFeatures[joiningForms[ci]] === feature.tag) {
-                Typr.U._applySubs(glyphIds, ci, tab, lookupList)
+    if (enableLigatures) {
```

Comment:
I noticed this part was responsible for ligature substitution, and I moved it behind the flag, but if it inadvertently disables something else do let me know!

Reply:
Ah, yeah I think we need to be more selective -- some substitutions are required for correctness (e.g. in Arabic scripts). We'll need to filter the supportedFeatures to remove just the stylistic ligatures. I think that's `liga`, but I'd have to research the others.

Reply:
Documentation for the feature tags: https://learn.microsoft.com/en-us/typography/opentype/spec/features_ko#tag-liga